matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.css

```diff
@@ -475,6 +475,15 @@ td[name="contact"] {
   white-space: normal;
   line-height: 12px;
 }
+.xTooltip {
```

Comment:
This seems too generic and will likely break the intended behavior for other tooltips. Can you find a more specific selector that only applies to tooltips in the matches table?